src/Deprecated14/Stream.extension.st

```diff
@@ -29,3 +29,16 @@ Stream >> basicNextPutAll: aCollection [
 	self deprecated: 'Use #nextPutAll:' transformWith: '`@rcv basicNextPutAll: `@arg1' -> '`@rcv nextPutAll: `@arg1'.
 	^ self nextPutAll: aCollection
 ]
+
+{ #category : '*Deprecated14' }
```

Comment:
are you sure on this one? Because marcus told me that his idea did not work or need a bit more polishing like using double dispatch.
@MarcusDenker

Reply:
Yes, this is the reason the PR has a conflict. I'll fix the conflict later